vscode/src/completions/providers/unstable-fireworks.ts

```diff
@@ -36,6 +35,29 @@ const MODEL_MAP = {
     'llama-code-13b-instruct': 'fireworks/accounts/fireworks/models/llama-v2-13b-code-instruct',
 }
 
+function getContextWindowChars(model: string): number {
```

Comment:
Thoughts on making it type-safe?

```suggestion
type FireworksModel = keyof typeof MODEL_MAP

function getContextWindowChars(model: FireworksModel): number {
```